Description: customize nvpp, nsight wrappers and nvcc.profile
 adjust paths etc.
Author: Andreas Beckmann <anbe@debian.org>

--- a/nvidia-cuda/bin/nsight
+++ b/nvidia-cuda/bin/nsight
@@ -1,3 +1,6 @@
 #!/bin/sh
-CUDA_BIN=$( cd "$( dirname "$0" )" && pwd )
-PATH=$PATH:$CUDA_BIN LD_LIBRARY_PATH=$LD_LIBRARY_PATH:$CUDA_BIN/../lib:$CUDA_BIN/../lib64 UBUNTU_MENUPROXY=0 LIBOVERLAY_SCROLLBAR=0 $CUDA_BIN/../libnsight/nsight $@
+
+JAVA_HOME=#CUDA_JAVA_HOME#
+PATH=$JAVA_HOME/bin:$PATH
+
+UBUNTU_MENUPROXY=0 LIBOVERLAY_SCROLLBAR=0 exec /usr/lib/nvidia-nsight/nsight "$@"
--- a/nvidia-cuda/bin/nvcc.profile
+++ b/nvidia-cuda/bin/nvcc.profile
@@ -1,14 +1,17 @@
+# Syntax:
+#   name  = <text>   assignment
+#   name ?= <text>   conditional assignment
+#   name += <text>   prepend
+#   name =+ <text>   append
+# Predefined variables: _SPACE_, _HERE_, _TARGET_DIR_, _TARGET_SIZE_
 
-TOP              = $(_HERE_)/..
+NVVMIR_LIBRARY_DIR = /usr/lib/nvidia-cuda-toolkit/libdevice
 
-NVVMIR_LIBRARY_DIR = $(TOP)/nvvm/libdevice
+PATH		+= /usr/lib/nvidia-cuda-toolkit/bin:
 
-LD_LIBRARY_PATH += $(TOP)/lib:
-PATH            += $(TOP)/nvvm/bin:$(_HERE_):
+#INCLUDES	+=
 
-INCLUDES        +=  "-I$(TOP)/$(_TARGET_DIR_)/include" $(_SPACE_)
+LIBRARIES	=+ $(_SPACE_) -L/#LIBDIR#/stubs
 
-LIBRARIES        =+ $(_SPACE_) "-L$(TOP)/$(_TARGET_DIR_)/lib$(_TARGET_SIZE_)/stubs" "-L$(TOP)/$(_TARGET_DIR_)/lib$(_TARGET_SIZE_)"
-
-CUDAFE_FLAGS    +=
-PTXAS_FLAGS     +=
+#CUDAFE_FLAGS	+=
+#PTXAS_FLAGS	+=
--- a/nvidia-cuda/bin/nvvp
+++ b/nvidia-cuda/bin/nvvp
@@ -1,3 +1,6 @@
 #!/bin/sh
-CUDA_BIN=$( cd "$( dirname "$0" )" && pwd )
-PATH=$PATH:$CUDA_BIN LD_LIBRARY_PATH=$LD_LIBRARY_PATH:$CUDA_BIN/../lib:$CUDA_BIN/../lib64 UBUNTU_MENUPROXY=0 LIBOVERLAY_SCROLLBAR=0 $CUDA_BIN/../libnvvp/nvvp $@
+
+JAVA_HOME=#CUDA_JAVA_HOME#
+PATH=$JAVA_HOME/bin:$PATH
+
+UBUNTU_MENUPROXY=0 LIBOVERLAY_SCROLLBAR=0 exec /usr/lib/nvidia-visual-profiler/nvvp "$@"
